### DISCUSSION TITLE
Asker ve Muzlar

### DISCUSSION THREAD

**Discussion by yahyaemiraksoy220** [sonsuzus/sonsuzus#490] [Soru Cevap]
Bir asker dükkandan w adet muz satın almak istiyor . İlk muz için k dolar, ikinci muz için 2 k dolar, vb. ödemesi gerekiyor (başka bir deyişle, i'inci muz için i · k dolar ödemesi gerekiyor ).

n doları var . Arkadaşı askerden w tane muz alabilmek için kaç dolar borç alması gerekiyor ?

Giriş
İlk satırda üç pozitif tam sayı k ,  n ,  w ( 1 ≤   k ,  w   ≤ 1000 , 0 ≤  n  ≤ 10 9 ), ilk muzun maliyeti, askerin başlangıçta sahip olduğu dolar miktarı ve istediği muz sayısı bulunmaktadır.

Çıktı
Bir tam sayı çıktısı alın — askerin arkadaşından ödünç alması gereken dolar miktarı. Eğer ödünç alması gerekmiyorsa, 0 çıktısı alın .
`odunc_alinacak_para=0
k=int(input("bir muz kaç dolar"))
if k<=1:
    while k<=1:
        k=int(input("bir muz kaç dolar"))
n=int(input("askerin başlangıçtga kaç doları var"))
if n<=0 and n>=109:
    while n<=0 and n>=109:
        n=int(input("askerin başlangıçtga kaç doları var"))
w=int(input("asker kaç tane muz almalı"))
if w>=1000:
    while w>=1000:
        w=int(input("asker kaç tane muz almalı"))
odunc_alinacak_para=k*w-n
print("askerin",odunc_alinacak_para,"kadar para ödünç alması lazım")`

**Comment by sonsuzus** [comment]
Programı cevap kısmına yazıp code bloğu içine alman gerekiyor. Böyle anlaşılmıyor.
